uber/server.py

```diff
@@ -101,11 +101,17 @@ def index(self):
 cherrypy.tree.mount(Root(), c.PATH, c.APPCONF)
 static_overrides(join(c.MODULE_ROOT, 'static'))
 
-DaemonTask(check_unassigned, interval=300,          name="mail unassg")
-DaemonTask(detect_duplicates, interval=300,         name="mail dupes")
-DaemonTask(check_placeholders, interval=300,        name="mail placeh")
 
-DaemonTask(SendAllAutomatedEmailsJob.send_all_emails, interval=300,   name="send emails")
+def reg_checks():
```

Comment:
Super minor nitpick: I'd have named this `maintenance_report_checks` or something a little more specific.